docs/guides/simulate-with-qiskit-sdk-primitives.mdx

```diff
@@ -15,13 +15,8 @@ The following sections show how to use the reference primitives to run your work
 
 ## Use the reference `Estimator`
```

Comment:
This does not render well, at least not on my browser: 

<img width="361" alt="image" src="https://github.com/user-attachments/assets/9f99ad24-ac2c-40f0-8bba-2e34fa911170">

Reply:
This is fixed in prod, but the issue has not yet been fixed in previews.  It's safe to ignore.